1_html-css/cv.html

```diff
@@ -0,0 +1,89 @@
+  </head>
+  <body>
+    <header>
+      <div id="header-container">
+        <h1>Shahnawaz Wardak</h1>
+        <p>Trainee developer</p>
+        <nav>
+          <ul>
+            <li><a href="#about-me">About</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section role="region" aria-labelledby="about-me">
+        <h2 id="about-me">About me</h2>
+        <div class="info">
+          <img
+            id="image"
+            src="https://images.ctfassets.net/oq23oflwrts4/3gHmjeMRfGf0CKhPgGVGQC/796f1e754454a54c1366c312e5f5e82a/20211010_134018_04.jpg"
+          />
+          <p id="body">
+            Although my academic background thus far has been in marketing and
+            literature, I have come to the conclusion that now is the time to do
+            what I have always wanted to do, i.e. to become a developer. I have
+            previously worked in technical support and now I am in the process
+            of upskilling my self by learning to code.
+          </p>
+        </div>
+      </section>
+      <section role="region" aria-labelledby="portfolio">
+        <h2 id="portfolio">Portfolio</h2>
+        <p id="body">
+          Here is my growing list of the works that I have been doing
+        </p>
+
+        <div class="portfolio">
+          <div class="form">
+            <img
+              class="port-img"
+              src="https://images.ctfassets.net/oq23oflwrts4/3mVP84ahOKF7i0P2bedenX/4fbe0ee2e8398eed5fc705c3bb84175d/Screenshot_2023-08-19_at_12.37.31.png"
+            />
+            <p id="links">
+              <a
+                href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form"
+                >Survey form
+              </a>
+            </p>
+          </div>
+          <div class="nutrition">
+            <img
+              class="port-img"
+              src="https://images.ctfassets.net/oq23oflwrts4/2eF5entSVDOEpayKqiWV4n/ec479d822c23a50009a16b0c73c037b9/Screenshot_2023-08-19_at_13.48.11.png"
+            />
+            <p id="links">
+              <a
+                href="https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-typography-by-building-a-nutrition-label/step-67"
+              >
+                Nutritional label
+              </a>
+            </p>
+          </div>
+        </div>
+      </section>
+      <section>
+        <h2 id="contact">Contact</h2>
+        If you need assistance with your projects, please send me an
+        <a href="seewardak@hotmail.com">email </a> with all relevant
+        information. Alternatively, feel free to contact me via:
+        <div id="SoMe">
+          <a href="facebook.com">
+            <img src="./Facebook.png" />
+          </a>
+          <a href="linked.com"> <img src="./Linkedin.png" /> </a>
```

Comment:
The images are very big in comparison to other elements in the page. Might be a good idea to adjust their displayed size! 💡

Reply:
I have changed the image sizes, as requested.